avm/res/network/network-manager/connectivity-configuration/main.bicep

```
@@ -49,12 +41,20 @@ resource connectivityConfiguration 'Microsoft.Network/networkManagers/connectivi
   name: name
   parent: networkManager
   properties: {
-    appliesToGroups: appliesToGroups
+    appliesToGroups: map(
+      appliesToGroups,
+      (group) => {
+        groupConnectivity: group.groupConnectivity
+        isGlobal: string(group.isGlobal) ?? 'false'
+        networkGroupId: any(group.networkGroupId)
+        useHubGateway: string(group.useHubGateway) ?? 'false'
+      }
+    )
     connectivityTopology: connectivityTopology
-    deleteExistingPeering: connectivityTopology == 'HubAndSpoke' ? deleteExistingPeering : 'False'
+    deleteExistingPeering: connectivityTopology == 'HubAndSpoke' ? string(deleteExistingPeering) : 'false'
     description: description ?? ''
     hubs: connectivityTopology == 'HubAndSpoke' ? hubs : []
-    isGlobal: isGlobal
+    isGlobal: string(isGlobal) ?? 'false'
```

```suggestion
    isGlobal: string(isGlobal)
```

Would that not be enough with the default being `false` anyways?

---

Yes it should. Thanks